solr/core/src/test/org/apache/solr/handler/designer/ManagedSchemaDiffTest.java

```diff
@@ -40,7 +40,6 @@ public class ManagedSchemaDiffTest extends SolrCloudTestCase {
 
   @BeforeClass
   public static void createCluster() throws Exception {
```

Comment:
```suggestion
  public static void createCluster() throws Exception {
    System.setProperty("managed.schema.mutable", "true");
```
to undo accidental removal on the next line.